railties/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Send 303 see other status code back for the destroy action on newly generated
```

Comment:
Capitalizing "See Other" is a bit easier to read I think. Maybe also the [official](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/303) notation?
```suggestion
*   Send 303 See Other status code back for the destroy action on newly generated
```